scripts/operations/etcd/take-etcd-manual-backups.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
This file does not report as executable despite being invoked like it is. Please double-check that this has the executable bit set.